src/components/ScreenMain/CharacterConfigModalContent/index.tsx

```diff
@@ -0,0 +1,97 @@
+import { useContext } from "react";
+import styled from "styled-components";
+import { GameContext } from "../../GameProvider";
+import ClassText from "../Footer/ClassText";
+import FactionText from "../Footer/FactionText";
+
+export interface Props {
+  onClose?: () => void;
+}
+
+const CharacterConfigModalContent = (props: Props) => {
+  const { character, levelUp } = useContext(GameContext);
+  return (
+    <ModalMain>
+      <InfoSection>
+        <div>
+          <FactionText faction={character.faction} /> <ClassText heroClass={character.heroClass} />
+        </div>
+        <div>
+          <LevelText>Level {character.level}</LevelText>
+        </div>
+      </InfoSection>
+      <div style={{ display: "flex", alignItems: "center", justifyContent: "center", flex: "1" }}>
+        TODO stats per level table
+      </div>
+      <ModalFooter>
+        <ButtonsContainer>
+          <ActionButton
+            role="button"
+            onClick={() => {
+              levelUp();
```

Comment:
I think it would be a better experience if we disabled the button or hid it if the user was already at max level. I think being able to click level up and do nothing is a bit weird.  Can we set disabled to be true if level is equal to 5?